### PR TITLE
Implemented Frame Icon

### PR DESCRIPTION
**Pull request by SidaDan**
Hello :-)

I have another small contribution for you, hope you find this
usefull.

Frank

### CODE REVIEW

File: src/simplejavatexteditor/UI.java [hunk 1]
```diff
@@ -93,6 +93,14 @@
     private boolean edit = false;
 
     public UI() {
+         try {
+            ImageIcon image = new ImageIcon("icons/ste.png");
+            super.setIconImage(image.getImage());
+        } // try
```
Comment (pH-7) [inline src/simplejavatexteditor/UI.java]: Is there a reason..., or can you remove the `// try ` comment?

**Comment by pH-7** [comment]
Awesome @SidaDan 💯 
Can you please just review my above comment?

**Comment by SidaDan** [comment]
Hi,

yes, you can remove it.

Greeting Frank

**Comment by pH-7** [comment]
Thanks very much @SidaDan 
See you next time hopefully 😃 

Have a wonderful day!!

**Pull request merged**